ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -690,24 +694,29 @@ static void HandleKernelRequest(void* messageMemory, uint32_t messageSize)
 #endif
         }
     }
+    else
+    {
+        absolutePath = request.path;
```

Comment:
Under what cases would we expect a request.path?  (FileOps?)

Reply:
Yep, FileOps will always send a path along.

Reply:
It would be great if we can add a small comment where appropriate in this function
"FileOp operations contain full path.  Vnode operations must look up full path from fsid"

Reply:
Done in this latest version.